all_repos/push/gitlab_pull_request.py

```diff
@@ -31,14 +31,26 @@ def push(settings: Settings, branch_name: str) -> None:
     }
 
     remote_url = git.remote('.')
-    _, _, repo_slug = remote_url.rpartition(':')
-    repo_slug = _strip_trailing_dot_git(repo_slug)
+    parsed_url = urllib.parse.urlparse(remote_url)
+    repo_slug = _strip_trailing_dot_git(parsed_url.path[1:])
     repo_slug = urllib.parse.quote(repo_slug, safe='')
```

Comment:
I don't think this logic should have changed

Reply:
The original code produced a `repo_slug` that is `//gitlab.com/foo/bar` but we need `foo/bar` (quoting removed for clarity).
See [gitlab API docs](https://docs.gitlab.com/ee/api/merge_requests.html#create-mr), the so called slug can either be an `id` or a [`NAMESPACE/PROJECT_PATH`](https://docs.gitlab.com/ee/user/group/#namespaces).

Also the original code did not work for me.

Reply:
it shouldn't do that if they are ssh urls?

Reply:
Ah, I see, I'll fix that.

Reply:
Now only ssh is used.